### ISSUE TITLE
Posts are failing to load

### ISSUE THREAD

**Issue by jessiemay1993** [Wikidepia/InstaFix#26]
When providing a link to an Instagram reel, link will try to load the reel, but will ultimately fail. 
![InstaFix](https://github.com/Wikidepia/InstaFix/assets/139272043/71ef82ff-35c4-4352-b6be-63bf69893f43)
![InstaFix2](https://github.com/Wikidepia/InstaFix/assets/139272043/f542ba8e-1e8c-4958-8382-4d3825f661ef)

I have also found in some cases it won't even attempt to load a reel at all even though the service says that it is currently running.

**Comment by ianw11** [comment]
I am also seeing reels failing to embed (into Discord)

**Comment by Wikidepia** [comment]
Yea, it is #25 I will try to find solution today

**Comment by FionaShine** [comment]
Have a same problem. Posts are failing to load, appears just a link. Discord and Instagram was updates, but problem continue.

**Comment by rafitamolin** [comment]
@Wikidepia ddinstagram.com as of today just throws a ERR_CONNECTION_REFUSED error

**Comment by Wikidepia** [comment]
> @Wikidepia ddinstagram.com as of today just throws a ERR_CONNECTION_REFUSED error

Should be fixed, I forgot to start caddy. Thanks for the report!

**Comment by AccountNameIsTaken** [comment]
+1, also seeing this issue. Thanks so much for your hard work on this! I really appreciate the embed preventing me from having to go to that abysmal site!

**Comment by alzedd** [comment]
Still getting connection refused today

**Comment by Noshtzy** [comment]
It's dead Jim

**Comment by GeorgeWL** [comment]
Now seeking that the text embeds, but the images do not

**Comment by kpopdev** [comment]
@Wikidepia this is a video but it shows a image https://www.ddinstagram.com/p/CukuY42N70s/

**Comment by sirkostya009** [comment]
I believe the issue of absent previews could be related to long post texts. Try limiting the amount of chars a string can have in a preview.

**Comment by sirkostya009** [comment]
@Wikidepia I believe the issue with absent media preciews could be related to too long texts. Try limiting the amount of chars a preview can have. Cause every post with a gazilion of hastags doesnt have video while those with just one sentence do.

**Comment by Wikidepia** [comment]
> @Wikidepia I believe the issue with absent media preciews could be related to too long texts. Try limiting the amount of chars a preview can have. Cause every post with a gazilion of hastags doesnt have video while those with just one sentence do.

Yeah, that might be it. I forgot to add description limit to image post. Thanks!

**Comment by FionaShine** [comment]
I'm not sure if this is related, today the problem recurred. Thanks for InstaFix!

**Comment by sirkostya009** [comment]
@FionaShine is there a chance you had an igshid param in the link? @Wikidepia sometimes links with igshid param don't have previews as well. Like this one: https://www.ddinstagram.com/reel/Cu9xZyRN3db/?igshid=MzRlODBiNWFlZA==. And removing the param doesn't help either.

**Comment by Wikidepia** [comment]
Video embed is a bit fragile at the moment, i am still finding solution

**Comment by azgoodaz** [comment]
![Screenshot 2023-08-25 081111](https://github.com/Wikidepia/InstaFix/assets/10385818/6d02c86a-ba72-4dbe-8bf0-420ca216d705)

**Comment by alexjyong** [comment]
Not too sure what is going on. 

<img width="716" alt="image" src="https://github.com/Wikidepia/InstaFix/assets/5464404/2405415f-0017-49fa-968f-7b7ea1a73337">
Reels just hang on my dev instance (with no code changes)

standard photos work with my local server, but not with ddinstagram

<img width="682" alt="image" src="https://github.com/Wikidepia/InstaFix/assets/5464404/4c671274-f45d-4d73-865c-5f0817f2cf30">

**Comment by Wikidepia** [comment]
> standard photos work with my local server, but not with ddinstagram

Not sure why but sometimes Instagram showed post not found when it is actually exist

> Reels just hang on my dev instance (with no code changes)

IG rate limited speed for videos so it will hang for a while, you can use proxy like https://github.com/Wikidepia/InstaFix-proxy to bypass it

**Comment by itsmostafa75** [comment]
> > standard photos work with my local server, but not with ddinstagram
> 
> Not sure why but sometimes Instagram showed post not found when it is actually exist
> 
> > Reels just hang on my dev instance (with no code changes)
> 
> IG rate limited speed for videos so it will hang for a while, you can use proxy like https://github.com/Wikidepia/InstaFix-proxy to bypass it

I have the same problem and the Instagram post could not be downloaded in Telegram. How should we use this proxy that you introduced? Can we normal users also use this feature or do we need a personal server?
